lib/absinthe/phase/document/complexity.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Absinthe.Phase.Document.Complexity do
+  @moduledoc false
+
+  # Analyses document complexity.
+
+  alias Absinthe.{Blueprint, Phase}
+
+  use Absinthe.Phase
+
+  @default_complexity 1
```

Comment:
I have allowed `0` because a field with no children will have a child complexity of 0 passed to it.